README.md

```diff
@@ -70,15 +70,18 @@ If you install LuaRocks for use with SILE via `pacman`, use the `lua51-*` varian
 
 #### Fedora
 
-A [COPR][copr] repository is available for Fedora users with packages of SILE and all the necessary dependencies.
-Fedora 38 and later are supported.
-There is work in progress to get the packages added to the official Fedora repository.
+Fedora Linux have SILE in their official Fedora repository. To install run:
```

Comment:
Plural is weird. Was "Fedora Linux users" intended (from the previous wording)?

Reply:
```suggestion
Fedora Linux has SILE in their official repositories. To install run:
```